### DISCUSSION TITLE
How to convert from an AD<double>  to a double ?

### DISCUSSION THREAD

**Discussion by abczhou123** [coin-or/CppAD#133] [Q&A]
eg: 
AD<double> a=10;
double b=a;//error

**Comment by bradbell** [comment]
see
https://coin-or.github.io/CppAD/doc/value.htm

and
https://coin-or.github.io/CppAD/doc/var2par.htm

**Comment by abczhou123** [comment]
> 见 https://coin-or.github.io/CppAD/doc/value.htm
> 
> 和 https://coin-or.github.io/CppAD/doc/var2par.htm

Thank you.

**Comment by bradbell** [comment]
I am converting this from issue 132 to a discussion
